xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.cpp

```diff
@@ -24,15 +24,35 @@
 #include <va/va_drmcommon.h>
 #include <drm_fourcc.h>
 #include "utils/log.h"
+#include "utils/EGLUtils.h"
 
 using namespace VAAPI;
 
-void CVaapiTexture::Init(InteropInfo &interop)
+void CVaapiTexture::TestInterop(VADisplay vaDpy, EGLDisplay eglDisplay, bool& general, bool& hevc)
+{
+  general = hevc = false;
+#if KODI_HAVE_VAAPI2TEXTURE
+  CVaapi2Texture::TestInterop(vaDpy, eglDisplay, general, hevc);
+  CLog::Log(LOGDEBUG, "Vaapi2 EGL interop test results: general %s, hevc %s", general ? "yes" : "no", hevc ? "yes" : "no");
```

Comment:
those logs just generate log spam. drop them here. better log at the location it gets registered.

Reply:
Where it's registered we don't know whether it's vaapi1 or vaapi2 (since it does not matter for registering)

Reply:
same rule as for configure. don't mix v1 and v2. register is done here:
CRendererVAAPI::Register, for all methods.

Reply:
done